algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -21,25 +21,25 @@ class CmsLaws extends PropSpec with PropertyChecks with Matchers {
   }
 
   property("CountMinSketch[Short] is a Monoid") {
-    implicit val cmsMonoid = CMS.monoid[Short](EPS, DELTA, SEED)
+    implicit val cmsMonoid = new CMSMonoid[Short](EPS, DELTA, SEED, 0)
```

Comment:
Can we use a const for `0` to explain the magic number, just like we do for EPS, DELTA, and SEED?